truss/build.py

```diff
@@ -37,6 +38,15 @@ def mk_truss(
         TrussHandle: A handle to the generated Truss that provides easy access to content inside.
     """
     model_framework = model_framework_from_model(model)
+    if (model_framework.typ() == ModelFrameworkType.XGBOOST):
+        click.echo(
+            click.style
+            (
+                '''WARNING: Truss uses XGBoost save/load which has a
+                different interface during inference than the class
+                you used to train this model.''', fg='yellow'
```

Comment:
Good message but it should include a link to some sort of XGBoost docs (or our docs) explaining the difference in detail